src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,182 @@
+// React Imports
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+// Solid Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Material UI Imports
+import AccountCircle from '@mui/icons-material/AccountCircle';
+import AppBar from '@mui/material/AppBar';
+import Badge from '@mui/material/Badge';
+import Box from '@mui/material/Box';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import MenuIcon from '@mui/icons-material/Menu';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+import Tab from '@mui/material/Tab';
+import Tabs from '@mui/material/Tabs';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+// Custom Component Imports
+import NavMenu from './NavMenu';
+import NavMenuMobile from './NavMenuMobile';
+
+/**
+ * NavBar Component - Component that generates NavBar section for PASS
+ *
+ * @memberof GlobalComponents
+ * @name NavBar
+ */
+
+const NavBar = () => {
+  const { session } = useSession();
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [anchorElMobile, setAnchorElMobile] = useState(null);
+  const [anchorElNotifications, setAnchorElNotifications] = useState(null);
+  const [openMenu, setOpenMenu] = useState(false);
+  const [openMobileMenu, setOpenMobileMenu] = useState(false);
+  const [value, setValue] = useState('home');
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  const menuId = 'primary-search-account-menu';
+  const mobileMenuId = 'primary-search-account-menu-mobile';
+
+  const handleOpenMenu = (event) => {
+    setAnchorEl(event.currentTarget);
+    setOpenMenu(true);
+  };
+
+  const handleNotificationsMenu = (event) => {
+    setAnchorElNotifications(event.currentTarget);
+  };
+
+  const handleNotificationsClose = () => {
+    setAnchorElNotifications(null);
+  };
+
+  const handleOpenMobileMenu = (event) => {
+    setAnchorElMobile(event.currentTarget);
+    setOpenMobileMenu(true);
+  };
+
+  return (
+    <Box sx={{ flexGrow: 1 }}>
+      <AppBar position="static" color="primary">
+        <Toolbar>
+          {session.info.isLoggedIn ? (
+            <>
+              <Typography variant="h4" noWrap component="div" mr="10px">
+                PASS
+              </Typography>
+              <>
+                <Box>
+                  <Tabs
```

Comment:
Seems like we have a minor issue with the Tabs.

https://github.com/codeforpdx/PASS/assets/14917816/9477ed52-c393-4cce-aa7f-c5bf133052af

At first, I thought the text colors were like that because we're only active on one page. However, as you see from the clip, the colors doesn't change when you go to a different page. Either keep the colors uniformed, or make it change color based on the page the user's using.

Reply:
I haven't been able to determine what's causing that. If you click it a second time then it highlights properly. And it only occurs on the Forms tab, not Home.

Reply:
Fixed this issue by having the NavBar on a higher level inside index.jsx. This would prevent any potential re-rendering of NavBar, which I think is likely the cause of this problem.